.github/workflows/typescript-build.yml

```
@@ -0,0 +1,26 @@
+name: TypeScript Build
+
+on:
+  push:
+    branches-ignore: [gh-pages]
+    paths-ignore: ['**.md']
+  pull_request:
+    branches-ignore: [gh-pages]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '#skip-build')"
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm ci
+      - name: TypeScript Build Test
+        run: npm run build --noEmit
```

There would be no problem with emitting.

```suggestion
        run: npm run build
```